src/SUMMARY.md

```diff
@@ -12,11 +13,22 @@
   - [Applicator](tools/applicator.md)
   - [Outliner](tools/outliner.md)
 
+- [Essential Features](chapter_2.md)
+  - [Catalog](essentials/catalog.md)
+  - [Radial Menus](essentials/radial_menus.md)
+  - [Presets](essentials/presets.md)
+  - [Quickbar](essentials/quickbar.md)
+
+- [Components](chapter_3.md)

```

Comment:
for all of these, it seems useless to copy their settings and tooltips to the book - someone will have to constantly update this as we update the game and it gives the user no new information

Reply:
instead we could be giving users tips on how to use the components more effectively or something like that